package.json

```diff
@@ -71,6 +71,7 @@
     "gulp-watch": "^4.3.5",
     "jest": "^19.0.2",
     "mock-stdin": "^0.3.0",
+    "sinon": "^1.17.7",
```

Comment:
Ok, didn't know this. I'll change this. Can you give me a hint how to handle the problem described in 3.?